kevinm23nelson/Travel-Tracker
# Iteration 5 accessibility/general refactoring

What does this PR do?
This PR implements a function to display pending trips in the UI, showing either a message if there are no pending trips or details of the pending trips along with a relevant image and destination overlay.

Where should the reviewer start?
The reviewer should start by examining the displayPendingTrips function in the relevant JavaScript file to understand how pending trips data is being processed and displayed in the DOM.

How should this be manually tested?
Manually test this by loading the application and verifying the pending trips section. Ensure that the text and image update correctly when there are no pending trips and when there are one or more pending trips.

Any background context you want to provide?
This function is part of a larger project that displays trip information for users. The displayPendingTrips function specifically handles the display logic for trips that are in a pending status, ensuring that users are informed about their pending trips or the absence thereof.
